.github/workflows/suppressions-test.yaml

```diff
@@ -6,9 +6,6 @@ on:
       - main
       - typespec-next
   pull_request:
```

Comment:
PR triggers should run on all branches.  I can't remember why I added these filters in the first place.  Maybe just defense-in-depth to avoid problems if the check was flaky?